pepper-apis/src/main/java/org/broadinstitute/ddp/service/PdfGenerationService.java

```diff
@@ -141,10 +142,21 @@ public InputStream generateFlattenedPdfForConfiguration(PdfConfiguration configu
         Participant participant = loadParticipantData(handle, configuration, userGuid);
         Map<Long, ActivityResponse> instances = loadActivityInstanceData(handle, configuration, participant);
         Long languageId = null;
+        List<Long> templateIds = Collections.emptyList();
         if (participant.getUser().getProfile() != null) {
             languageId = participant.getUser().getProfile().getPreferredLangId();
+            if (languageId != null) {
+                //make sure pdf templates exist for this language else fallback to study default language
+                templateIds = handle.attach(PdfDao.class)
+                        .findTemplateIdsByVersionIdAndLanguageCodeId(configuration.getVersion().getId(), languageId);
+                if (templateIds.isEmpty()) {
+                    LOG.warn("User : {} has language {} as preferred language but NO PDF templates found for the language. " 
+                            + "Using study default language ", userGuid, participant.getUser().getProfile().getPreferredLangCode());
+                }
+            }
         }
-        if (languageId == null) {
+
+        if (languageId == null || templateIds.isEmpty()) {
             //use study default language if exists
```

Comment:
Expectation here is that if  a study has NON 'english' as default language, pdf templates exist for this language. 
We might want to have this check in study builder if we are concerned about this expectation.